.gitignore

```diff
@@ -0,0 +1,16 @@
+
+.cache/
+coverage/
+dist/*
+!dist/index.html
+node_modules/
+*.log
+
+# OS generated files
+.DS_Store
```

Comment:
I see that the `.gitignore` file is copied from somewhere - and it is ok with me, but even if it is copied, removing things that you don't need for sure is a good practice. So my question is, are you sure that you need all those things mentioned in the `.gitignore` file 😄 ?